### PR TITLE
Getting started docs updates

### PR DESCRIPTION
**Pull request by ColinEberhardt**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -4,46 +4,50 @@ layout: default
...
-        new Notification('Hello World');
+        const status = document.getElementById('status')
+        status.innerText = 'ready';
       });
   </script>
 </body>
 ```
 
-This file uses the ssf API to handle the container `ready` promise, then creates a notification. Notice that it loads the `containerjs-api.js` script, which provides the client-side APIs (within OpenFin).
+The above displays a simple welcome message and indicates the status of the ContaunerJS APIs. The `containerjs-bundle.js` is the client-side bundle that detects the container that the page is running within (e.g. Electron, OpenFin) and provides the required API implementation. In the above code, this bundle is being downloaded from the `unpkg` CDN for simplicity.
```
Comment (BenLambertNcl) [inline docs/index.md]: ContaunerJS => ContainerJS

File: docs/index.md [hunk 2]
```diff
@@ -4,46 +4,50 @@ layout: default
...
+
+This file uses the ssf API to handle the container `ready` promise, updating the status text when this lifecycle event occurs..
```
Comment (BenLambertNcl) [inline docs/index.md]: Only 1 full stop here

**Review by BenLambertNcl — APPROVED** [review]

File: docs/index.md (hunk 1 above)
Comment (ColinEberhardt) [reply docs/index.md]: Heh - good spot!

**Comment by ColinEberhardt** [comment]
Thanks for the review - fixed those issues.

**Pull request merged**